src/RobotCharacter.js

```diff
@@ -11,13 +11,11 @@ type RobotCharacterProps = {
 
 export default class RobotCharacter extends React.Component<RobotCharacterProps, {}> {
     render() {
-        const characterRadius = this.props.width / 2;
         const characterWidth = this.props.width * 0.75;
```

Comment:
We can get rid of this calculation and use `this.props.width` directly for the size of the robot as we don't have the circle anymore.

Reply:
Setting this.props.width as width and height gives this result
<img width="1228" alt="Screen Shot 2020-09-22 at 3 55 11 PM" src="https://user-images.githubusercontent.com/22668696/93930942-1a2f2480-fcec-11ea-97f5-620cf81acc38.png">

Reply:
Did you also update the x and y?

Reply:
Ah right, setting correct x and y set the robot character to the center; though, I remembered chat with Sepi, and she said "I think the smaller version looked better"

Reply:
In that case, if we want the robot smaller, let's control it with the 0.8 size factor in the Scene, rather than having 2 numbers that are multiplied.

Reply:
That is: change the 0.8 in Scene and remove this extra factor from RobotCharacter.

Reply:
Sounds good :)